src/main/org/firebirdsql/jdbc/AbstractResultSet.java

```diff
@@ -198,6 +201,7 @@ public AbstractResultSet(RowDescriptor rowDescriptor, List<RowValue> rows,
         this.rowDescriptor = rowDescriptor;
         fields = new FBField[rowDescriptor.getCount()];
         colNames = new HashMap<>(rowDescriptor.getCount(), 1);
+        closeableFields = null;
```

Comment:
Unnecessary initalization, fields are always initialized to null if not explicitly assigned